index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>Main</title>
+</head>
+
+<body>
+  <header>
+    <div class="heading">
+      Welcome
+    </div>
```

Comment:
- [ ] Nice to have logo on the left side however, logo has a link (a tag), please use h tag for the logo.